csympy/lib/csympy_wrapper.pyx.in

```diff
@@ -6,6 +6,12 @@ from libcpp.string cimport string
 from cpython cimport PyObject, Py_XINCREF, Py_XDECREF, \
     PyObject_CallMethodObjArgs
 
+DEF HAVE_SAGE = ${HAVE_SAGE}
+have_sage = ${HAVE_SAGE}
```

Comment:
Yeah, that's a good idea. We could also use `setup.py` to do it, if cythonizing and building the extension logic is shifted to `setup.py`. http://stackoverflow.com/a/26234748